README.md

```diff
@@ -1,14 +1,14 @@
 # DiskDict
 
 ## Description
-`DiskDict` is a file based module which stores data as key, value pairs like a python dictionary.
+`DiskDict` is a file based module which stores data as key, value pairs like a python dictionary. `DiskDict` stores the data on the disk unlike python dictionary which stores data in the RAM.
 
 ## Installation
 
-> Prerequisites: Python
+> Prerequisites: Python3
```

Comment:
@RamanjaneyuluIdavalapati So far `diskdict` is Py2 compatible and this pull request is making it required Py3? If yes, we should preserve Py2 compatibility or we risk breaking downstream software depending on this module.

Reply:
@prashanthellina My bad. It is also compatible for Py2. I don't know why I have changed this. Will give another commit by keeping it as `Python`